azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -602,9 +602,8 @@ private void doRun() {
     }
 
     final int attemptNo = this.node.getAttempt();
-    logInfo("Finishing job " + this.jobId + " attempt: " + attemptNo + " at "
+    logInfo("Finishing job " + this.jobId + (this.node.getAttempt() > 0 ? (" retry: " + attemptNo) : "") + " at "
```

Comment:
Why not use attemptNo instead of another call to getAttempt?

Even better, consider using a private method to calculate this string. This logic is in two places.